1_space_odyssey/index.html

```diff
@@ -10,7 +10,7 @@
 </head>
 <body class="fade-in">
     
-    <script src="./js/mannuel-9000/app.js"></script>
+    <script src="./js/mannuel-9000/mathias-9000/script.js"></script>
```

Comment:
please adjust your folder structure so that your path looks like this: `src="./js/mathias-9000/app.js"`